dashboard/UI-Test/test.yaml

```diff
@@ -0,0 +1,65 @@
+sequence:
+- testcase 1
+- testcase 2
+test:
+- testcase 1:
+  - scenario: invalid pbench-dashboard login
+  - step 1:
+      action: goto
+      target: http://10.1.170.201
+      targets:
+      - http://10.1.170.201
```

Comment:
Do we need to specify both `targets` and `target`?

Reply:
In all the steps the two keys always seem to have the same value....